README.md

```diff
@@ -57,22 +57,24 @@ As an alternative if this does not solve the OOM issues, try increase the -Xmx a
     <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
-        <version>2.21.0</version>
+        <version>3.5.1</version>
         <configuration>
             <forkCount>1</forkCount>
             <reuseForks>false</reuseForks>
             <argLine>-Xmx2048m -XX:MaxPermSize=512m</argLine>
         </configuration>
     </plugin>
 
-(please note that the forkMode option is deprecated and you should use forkCount and reuseForks instead)
+(please note that the forkMode option is deprecated, you should use forkCount and reuseForks instead)
+
+(please note that -XX:MaxPermSize will not work for Java > 8)
```

Comment:
Since this PR moves to Java 11 should we just remove the reference to `xPermSize` entirely?

Reply:
done